src/entities/SecurityToken/Features.ts

```diff
@@ -193,4 +169,64 @@ export class Features extends SubModule {
 
     return procedure.prepare();
   };
+
+  /**
+   * Disable a feature
+   *
+   * @param feature feature to enable
+   * @param opts feature options
+   */
+  public disable = async (
+    args: { feature: Feature },
+    opts?: EnableOpts
+  ): Promise<TransactionQueue<DisableFeatureArgs>> => {
+    const { feature } = args;
+
+    const alreadyDisabled = !(await this.isEnabled({ feature }));
+
+    if (alreadyDisabled) {
+      throw new PolymathError({
+        code: ErrorCode.FatalError,
+        message: `Feature '${feature}' already disabled`,
+      });
+    }
+
+    const { symbol } = this.securityToken;
+    const moduleName = this.getModuleNameFromFeature(feature);
+    const procedure = new DisableFeature(
+      {
+        symbol,
+        moduleName,
+        ...opts,
+      },
+      this.context
+    );
+
+    return procedure.prepare();
+  };
+
+  private getModuleNameFromFeature = (feature: Feature): ModuleName => {
```

Comment:
Can't this just be an object where the keys are of type `Feature` and the values are of type `ModuleName`?

Reply:
We are keeping it as it is, because this way allow us to throw an error if the Feature is not supported.